src/nv_ingest/modules/sinks/message_broker_task_sink.py

```diff
@@ -48,7 +49,7 @@ def extract_data_frame(message: ControlMessage) -> Tuple[Any, Dict[str, Any]]:
     """
     try:
         with message.payload().mutable_dataframe() as mdf:
```

Comment:
Only because I am curious. How is `message.payload()` transferred to this code? Is it serialized behind the into (JSON, something else) and then transferred via Morpheus here? Some sort of shared memory? Something else?

Reply:
Payload is a MessageMeta object which, generally, wraps a cuDF dataframe object. Morpheus has some specialized hooks into cuDF for managing it on the c++ side.